### PR TITLE
Add rqlite -ca-cert option to specify trusted root CA certificate

### PR DESCRIPTION
**Pull request by zmedico**
The -ca-cert option allows the user to specify a trusted X.509 root CA
certificate as an alternative to the --insecure option.

### CODE REVIEW

File: cmd/rqlite/main.go [hunk 1]
```diff
@@ -156,9 +158,24 @@ func expvar(ctx *cli.Context, cmd, line string, argv *argT) error {
 
 func sendRequest(ctx *cli.Context, makeNewRequest func(string) (*http.Request, error), urlStr string, argv *argT) (*[]byte, error) {
 	url := urlStr
+	rootCAs := (*x509.CertPool)(nil)
```
Comment (otoolep) [inline cmd/rqlite/main.go]: Why not just do `var rootCAs *x509.CertPool`? That seems more natural.

File: cmd/rqlite/main.go [hunk 2]
```diff
@@ -23,6 +24,7 @@ type argT struct {
 	Prefix      string `cli:"P,prefix" usage:"rqlited HTTP URL prefix" dft:"/"`
 	Insecure    bool   `cli:"i,insecure" usage:"do not verify rqlited HTTPS certificate" dft:"false"`
 	Credentials string `cli:"u,user" usage:"set basic auth credentials in form username:password"`
+	CACert      string `cli:"ca-cert" usage:"path to trusted X.509 root CA certificate"`
```
Comment (otoolep) [inline cmd/rqlite/main.go]: Just so the style remains consistent, can you also add a single letter option i.e. `cli:c,ca-cert"`.

File: cmd/rqlite/main.go (hunk 1 above)
Comment (zmedico) [reply cmd/rqlite/main.go]: Yes, that's better. Updated.

File: cmd/rqlite/main.go (hunk 2 above)
Comment (zmedico) [reply cmd/rqlite/main.go]: Yes, added.

**Comment by otoolep** [comment]
Great, thanks for your contribution.

**Comment by otoolep** [comment]
@zmedico -- is this worth backporting to the v4 series, and issuing a new release? Others might find this useful in the CLI.

**Comment by zmedico** [comment]
Sure, a backport could be useful. Sent as https://github.com/rqlite/rqlite/pull/551.

**Pull request merged**